portfolio/src/main/webapp/index.html

```diff
@@ -108,16 +108,42 @@ <h4>Fun Fact Roulette</h4>
           </a>
           <span id="fact-container"></span>
         </div>
-        <h4>Leave me a comment: </h4>
+        <h4>
+          Leave me a comment 
+          <span id="comment-icon">
+            <i class="material-icons">insert_emoticon</i>
+          </span>
+        </h4>
         <form action="/data" method="POST">
-          <label for="username">Name: </label>
+          <label for="username">Name </label>
           <input id="username" type="text" name="username">
-          <label for="commentText">Comment: </label>
-          <textarea id="commentText" name="commentText"></textarea>
+          <label for="commentText">Comment </label>
+          <textarea id="commentText" name="commentText" class="materialize-textarea"></textarea>
           <br/>
-          <input type="submit">
+          <button class="btn waves-effect waves-light pink lighten-3" type="submit" name="submit">Submit
+            <i class="material-icons right">send</i>
+          </button>
         </form>
-        <div id="comments-list"></div>
+        <div id="comments-section" style="display:none">
+          <h5>My Comments: </h5>
+          <div id="commentsNo-container">
```

Comment:
nit: generally prefer using full words rather than shortenings for clarity, unless it's a shortening that you'd also be displaying to humans like bps (bits per second) is one we use a lot in networking.

same with the limitNo variables you have around too (I kept reading it as no limit)